packages/components/src/styles/list/list.stories.ts

```diff
@@ -29,370 +23,209 @@ export default {
-                <li>Duis amet cupidatat incididunt ipsum veniam sunt esse et et</li>
+              <sd-icon name="content/picture" library="global-resources"></sd-icon>
+              Unordered list level 2
+              <ul class="sd-list--icon sd-list">
                 <li>
-                  Dolor sit
-                  <ol>
-                    <li>Amet</li>
-                    <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-                    <li>
-                      Dolor sit
-                      <ol>
-                        <li>Amet</li>
-                        <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-                        <li>Quis nostrud exercitation</li>
-                      </ol>
-                    </li>
-                    <li>Quis nostrud exercitation</li>
-                  </ol>
+                  <sd-icon name="content/picture" library="global-resources"></sd-icon>
+                  Unordered list level 3
                 </li>
-              </ol>
-            </li>
-            <li>
-              Dolor sit
-              <ol>
-                <li>Amet</li>
-                <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-              </ol>
+              </ul>
             </li>
           </ul>
         </li>
         <li>
-          Lorem Ipsum
-          <ul>
-            <li>
-              Dolor sit
-              <ol>
-                <li>Amet</li>
-                <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-              </ol>
-            </li>
-            <li>
-              Dolor sit
-              <ol>
-                <li>Amet</li>
-                <li>
-                  Ut enim ad minim veniam, quis nostrud exercitation
-                  <ul>
-                    <li>
-                      Dolor sit
-                      <ul>
-                        <li>Amet</li>
-                        <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-                      </ul>
-                    </li>
-                  </ul>
-                </li>
-                <li>Ut enim ad minim veniam, quis nostrud exercitation</li>
-              </ol>
-            </li>
-          </ul>
+          <sd-icon name="content/picture" library="global-resources"></sd-icon>
+          Unordered list level 1
         </li>
-      </preset-type>`.replaceAll('preset-type', type);
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'slot',
-          name: 'default',
-          values: [
-            { title: 'ul', value: preset('ul') },
-            { title: 'ol', value: preset('ol') }
-          ]
-        }
-      },
-      args
-    });
-  }
+        <li>
+          <sd-icon name="content/picture" library="global-resources"></sd-icon>
+          Unordered list level 1
+        </li>
+      </ul>
+    </div>`
+};
+
+/**
+ * Use `sd-list--horizontal` modifiers to set horizontal orientations for Icon list groups.
```

Comment:
* Use `&--horizontal` modifiers to set horizontal orientations for list groups with content icons.

Reply:
Updated